airbyte-integrations/connectors/destination-timeplus/destination_timeplus/destination.py

```diff
@@ -0,0 +1,122 @@
+        apikey=config["apikey"]
+        if endpoint[-1]=='/':
+            endpoint=endpoint[0:len(endpoint)-1]
+        env = Environment().address(endpoint).apikey(apikey)
+        stream_list = Stream(env=env).list()
+        all_streams = {s.name for s in stream_list}
+
+        # only support "overwrite", "append"
+        for configured_stream in configured_catalog.streams:
+            is_overwrite = configured_stream.destination_sync_mode == DestinationSyncMode.overwrite
+            stream_exists = configured_stream.stream.name in all_streams
+            if is_overwrite and stream_exists:
+                # delete the existing stream
+                Stream(env=env).name(configured_stream.stream.name).get().delete()
+            if is_overwrite or not stream_exists:
+                # create a new stream
+                DestinationTimeplus.create_stream(env, configured_stream.stream)
+                
+        for message in input_messages:
+            if message.type == Type.STATE:
+                # Emitting a state message indicates that all records which came before it have been written to the destination. So we flush
+                # the queue to ensure writes happen, then output the state message to indicate it's safe to checkpoint state
+                yield message
+            elif message.type == Type.RECORD:
+                record = message.record
+                
+                # this code is to send data to a single-column stream
+                # Stream(env=env).name(record.stream).column("raw", "string").ingest(payload=record.data)
+
+                Stream(env=env).name(record.stream).ingest(payload=record.data, format="streaming")
+            else:
+                # ignore other message types for now
+                continue
+        pass
+
+    @staticmethod
+    def create_stream(env,stream: AirbyteStream):
+        # singlel-column stream
+        # Stream(env=env).name(stream.name).column('raw','string').create()
+        
+        tp_stream=Stream(env=env).name(stream.name.strip())
+        for name,v in stream.json_schema['properties'].items():
+            tp_stream.column(name.strip(),DestinationTimeplus.type_mapping(v))
+        tp_stream.create()
+
+    @staticmethod
+    def type_mapping(v) -> str:
+        airbyte_type=v['type']
+        if type(airbyte_type) is list:
+            for t in list(airbyte_type):
+                if t!='null': 
+                    type_def={'type':t}
+                    if t=='array': type_def['items']=v['items']
+                    return DestinationTimeplus.type_mapping(type_def)
+        if airbyte_type=='number':return 'float'
+        elif airbyte_type=='integer':return 'integer'
+        elif airbyte_type=='boolean':return 'bool'
+        elif airbyte_type=='object':return 'string'
+        elif airbyte_type=='array':
+            return f"array({DestinationTimeplus.type_mapping(v['items'])})"
+        else:return 'string'
```

Comment:
Can you format the code, this doesn't look it is following the PEP-8.

Reply:
I ran `Format Document` from vs code.